<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>üì¶ Enhanced Item Cost Calculator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTarget">
        <property name="text">
         <string>Target Enhanced Count:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="targetEnhancedSpinBox">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCurrent">
        <property name="text">
         <string>Current Enhanced Count:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="currentEnhancedSpinBox">
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelEnhancedPrice">
        <property name="text">
         <string>Enhanced Item Price (each):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="enhancedPriceSpinBox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>999999999.990000009536743</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelNormalPrice">
        <property name="text">
         <string>Normal Item Price (each):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="normalPriceSpinBox">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>999999999.990000009536743</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>Calculate Costs</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultsTitleLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>üîç Calculation Results:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultNeededEnhancedLabel">
      <property name="text">
       <string>Enhanced items needed: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultNeededNormalLabel">
      <property name="text">
       <string>Normal items needed to craft: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultCostEnhancedLabel">
      <property name="text">
       <string>Cost if buying Enhanced directly: $ -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="resultCostNormalLabel">
      <property name="text">
       <string>Cost if buying Normal and crafting: $ -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
     <item>
     <widget class="QLabel" name="recommendationLabel">
      <property name="font">
       <font>
        <weight>Bold</weight>
       </font>
      </property>
      <property name="text">
       <string>üí° Recommendation: -</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
